clarify browser support (#3623)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/about-cloud/browsers.md

---
title: "Supported browsers"
id: "browsers"
description: "dbt Cloud supports the latest browsers like Chrome and Firefox."
---

To have the best experience with dbt Cloud, we recommend using the latest versions of the following browsers:

- [Google Chrome](https://www.google.com/chrome/) 
- [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/)

If you don't use the latest version of a recommended browser or if you use a browser that isn't listed, some dbt Cloud features may not work as expected.

> Note: To improve your experience using dbt Cloud, we suggest that you turn off ad blockers.

Author: i added this since this is recommended for the ide [see here](https://docs.getdbt.com/docs/cloud/dbt-cloud-ide/develop-in-the-cloud)

---
title: "Supported browsers"
id: "browsers"
description: "dbt Cloud supports the latest browsers like Chrome and Firefox."
---

To have the best experience with dbt Cloud, we recommend using the latest versions of the following browsers:

- [Google Chrome](https://www.google.com/chrome/) &mdash; Latest version is fully supported in dbt Cloud
- [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/) &mdash; Latest version is fully supported in dbt Cloud
- [Apple Safari](https://www.apple.com/safari/) &mdash; Latest version support provided on a best-effort basis
- [Microsoft Edge](https://www.microsoft.com/en-us/edge?form=MA13FJ&exp=e00) &mdash; Latest version support provided on a best-effort basis

dbt Cloud provides two types of browser support:

- Fully supported &mdash; dbt Cloud is fully tested and supported on these browsers. Features display and work as intended.
- Best effort &mdash; You can access dbt Cloud on these browsers. Features may not display or work as intended.

> You can still access and use dbt Cloud even if you don't use the latest recommended browser or if your browser isn't listed, However, please note that some features may not display correctly as intended.

Reviewer: ```suggestion
You may still be able to access and use dbt Cloud even without using the latest recommended browser or an unlisted browser. However, some features might not display as intended.
```

> Note: To improve your experience using dbt Cloud, we suggest that you turn off ad blockers.

Reviewer: ```suggestion
:::note
To improve your experience using dbt Cloud, we suggest that you turn off ad blockers.
:::
```
